include/z64cutscene.h

```diff
@@ -4,6 +4,27 @@
 #include <ultra64.h>
 #include <unk.h>
 
+typedef struct {
+    /* 0x00 */ s8    continueFlag;
+    /* 0x01 */ s8    cameraRoll;
+    /* 0x02 */ u16   nextPointFrame;
+    /* 0x04 */ f32   viewAngle; // in degrees
+    /* 0x08 */ Vec3s pos;
+} CutsceneCameraPoint; // size = 0x10
+
+typedef struct {
+    /* 0x00 */ Vec3f at;
+    /* 0x0C */ Vec3f eye;
+    /* 0x18 */ s16 roll;
+    /* 0x1A */ s16 fov;
+} CutsceneCameraAngle; // size = 0x1C
+
+typedef struct {
+    /* 0x0 */ CutsceneCameraPoint* atPoints;
```

Comment:
```suggestion
    /* 0x00 */ Vec3f focalPoint;
    /* 0x0C */ Vec3f eye;
    /* 0x18 */ s16 roll;
    /* 0x1A */ s16 fov;
} CutsceneCameraAngle; // size = 0x1C

typedef struct {
    /* 0x0 */ CutsceneCameraPoint* focalPoints;
```
This is a discussion outside the scope of this PR which I will be bringing up when I document Camera, but MM's Camera currently uses `focalPoint` where OoT uses `at`. I personally like focalPoint better as its a clearer name but it's very much open to discussion. I'd keep it consistent with what MM has for now, and I can change it later during Camera docs if need be